tests/java/test5/README.md

```diff
@@ -0,0 +1 @@
+Test case with reachable and unreachable methods in different class in different
```

Comment:
missing text? What's the difference between this and test 4?

Reply:
yes missing text.
test4 and text5 both have method in different class located in different java file, but test4 are packed in the same package as TestFuzzer while test5 are not. Thus test5 will need additional import statement.